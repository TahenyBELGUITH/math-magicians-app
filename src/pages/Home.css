.description,
.header {
  background-color: rgb(30, 130, 30);
  border: 3px solid burlywood;
  width: 90%;
  padding: 2rem;
  color: #fff;
  margin: auto;
  font-size: 30px;
  line-height: 50px;
  text-align: center;
}

.header {
  margin-top: 3rem;
  border-bottom: 1px solid transparent;
}

.description {
  border-top: 1px solid transparent;
}
