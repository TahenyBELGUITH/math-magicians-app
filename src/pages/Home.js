import React from 'react';
import './Home.css';

export default function Home() {
  return (
    <div>
      <h1 className="header">Welcome to our page!</h1>
      <p className="description">
        Lorem ipsum dolor sit amet consectetur adipisicing elit. Maxime
        mollitia, m! Provident similique accusantiOfficiis iure rerum voluptates
        a cumque velit
        {' '}
      </p>
    </div>
  );
}
