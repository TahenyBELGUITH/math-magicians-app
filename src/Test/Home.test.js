import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../pages/Home';

it('it should match the home page', () => {
  const tree = renderer.create(<Home />).toJSON();
  expect(tree).toMatchSnapshot();
});
