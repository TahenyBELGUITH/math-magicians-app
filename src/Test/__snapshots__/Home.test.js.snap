// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it should match the home page 1`] = `
<div>
  <h1
    className="header"
  >
    Welcome to our page!
  </h1>
  <p
    className="description"
  >
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Maxime mollitia, m! Provident similique accusantiOfficiis iure rerum voluptates a cumque velit
     
  </p>
</div>
`;
